### PR TITLE
Fix image resizing in markdown preview

### LINKED ISSUE #5608: Images get squished in narrow problem statements
### Describe the bug

image

### To Reproduce

Open a problem statement with an image in it and reduce the width

### Expected behavior

The image should be resized while keeping it's aspect ratio

### What browsers are you seeing the problem on?

Chrome

### Additional context

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by heddendorp**
Fixes #5608

#### General

- [x] This is a tiny change and I don't have enough time during exam season to go through the entire list.

### Screenshots

image

Compare to the linked issue

### CODE REVIEW

**Comment by ge65cer** [comment]
I can not reproduce the issue you reported, on which page exactly does the image not resize correctly?

**Comment by heddendorp** [comment]
> @heddendorp I can not reproduce the issue you reported, on which page exactly does the image not resize correctly?

Sorry for the delay, exams and stuff. The page in question is https://artemis.ase.in.tum.de/courses/188/file-upload-exercises/7124/participate/6302734

File: src/main/webapp/content/scss/global.scss [hunk 1]
```diff
@@ -752,6 +752,7 @@ Course Info Bar
 
     img {
         max-width: 100%;
+        height: auto;
```
Comment (stefanwaldhauser) [inline src/main/webapp/content/scss/global.scss:755]: Good fix to keep the aspect ratio. Also the solution used here: https://html.com/attributes/img-height/

**Review by stefanwaldhauser — APPROVED** [review]

**Comment by ge65cer** [comment]
Thanks for the link, this allowed me to investigate a little further. It looks like the instructor added the image not via Markdown but with a custom HTML tag specifically setting a width and height:
`﻿`

Normally, when the width and height are not custom-defined (e.g., the HTML tags are rendered from Markdown), the image is sized correctly. I do not consider this a bug in Artemis and pledge to close the issue as wont-fix.

**Review by ge65cer — COMMENTED** [review]
Thanks for the link, this allowed me to investigate a little further. It looks like the instructor added the image not via Markdown but with a custom HTML tag specifically setting a width and height:
`﻿`

Normally, when the width and height are not custom-defined (e.g., the HTML tags are rendered from Markdown), the image is sized correctly. I do not consider this a bug in Artemis and plead to close the issue as wont-fix.

**Comment by ge65cer** [comment]
@krusche Please decide how to proceed with this PR (see my comment above).

**Pull request closed without merging**